pai-experimental/cdp.sol

```diff
@@ -7,16 +7,17 @@ pragma solidity 0.4.25;
 // import "./price_oracle.sol";
 // import "./pai_issuer.sol";
 
-import "github.com/seeplayerone/dapp-bin/pai-experimental/3rd/math.sol";
-import "github.com/seeplayerone/dapp-bin/pai-experimental/3rd/note.sol";
-import "github.com/seeplayerone/dapp-bin/library/template.sol";
-import "github.com/seeplayerone/dapp-bin/pai-experimental/liquidator.sol";
-import "github.com/seeplayerone/dapp-bin/pai-experimental/price_oracle.sol";
-import "github.com/seeplayerone/dapp-bin/pai-experimental/pai_issuer.sol";
+import "github.com/evilcc2018/dapp-bin/pai-experimental/3rd/math.sol";
+import "github.com/evilcc2018/dapp-bin/pai-experimental/3rd/note.sol";
+import "github.com/evilcc2018/dapp-bin/library/template.sol";
+import "github.com/evilcc2018/dapp-bin/pai-experimental/liquidator.sol";
+import "github.com/evilcc2018/dapp-bin/pai-experimental/price_oracle.sol";
+import "github.com/evilcc2018/dapp-bin/pai-experimental/pai_issuer.sol";
```

Comment:
should change the import to 'seeplayerone' when merging back.

Reply:
OK